agents/lakeshore372/LS372_agent.py

```diff
@@ -314,17 +326,36 @@ def start_acq(self, session, params=None):
                     # For data feed
                     data['data'][channel_str + '_T'] = temp_reading
                     data['data'][channel_str + '_R'] = res_reading
+                    session.app.publish_to_feed('temperatures', data)
+                    self.log.debug("{data}", data=session.data)
 
                     # For session.data
                     field_dict = {channel_str: {"T": temp_reading,
                                                 "R": res_reading,
                                                 "timestamp": current_time}}
-
                     session.data['fields'].update(field_dict)
 
-                session.app.publish_to_feed('temperatures', data)
-
-                self.log.debug("{data}", data=session.data)
+                    # Also queries control channel if enabled
+                    if self.control_chan_enabled:
+                        temp = self.module.get_temp(unit='kelvin', chan=0)
+                        res = self.module.get_temp(unit='ohms', chan=0)
+                        cur_time = time.time()
+                        data = {
+                            'timestamp': time.time(),
+                            'block_name': 'control_chan',
+                            'data': {
+                                'control_T': temp,
+                                'control_R': res
```

Comment:
I debated on whether I'd prefer this name or "Input A", which the Lakeshore defaults to calling this channel (via `INNAME? A`.) We aren't using the channel name feature of the 372 at all, except for grabbing the default channel names, but if you wanted to grab names like done above it'd be:
```
channel_str = self.module.channels[0].name.replace(' ', '_')
```

Reply:
If it's up for debate, I personally think the names 'control_T', and 'control_R' are ok, and I'm not super keen to change it if we don't have to, only because we've been using these for a while. But I can switch if you think that would be best.